src/main/resources/assets/nbttooltips/lang/en_us.json

```diff
@@ -1,4 +1,4 @@
 {
   "nbttooltips.keybind": "Show NBT",
-  "item.nbt_tags.nbttooltips": "NBT: %s"
-}
\ No newline at end of file
+  "item.nbt_tags.nbttooltips": "NBT: "
```

Comment:
May I ask why you removed the "%s"?

Reply:
In `TooltipChanger.java`, a `MutableText` instance is created and all of the text components that make up the NBT are just appended to that instead of ever using that translation string as a format string. So when I tested it out, there was a literal "%s" in the NBT tooltip. I have no idea how it ever worked before without that literal "%s" being there, but I'm guessing that maybe Minecraft changed the way that unused format arguments are handled or something. Or maybe I'm dumb and you did provide an argument for that `%s` in some other way that I don't know of?

Reply:
Thank you for the explanation :)